samples/Samples.HttpClientDriver/Program.cs

```diff
@@ -0,0 +1,220 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Core.Tools;
+
+namespace Samples.HttpClientDriver
+{
+    public static class Program
```

Comment:
Can you make sure this app also has `[LoaderOptimization(LoaderOptimization.MultiDomainHost)]`? We want to make sure people running IIS (which this LoaderOptimization simulates) will get traces for each call. Or perhaps that could be done as a follow-up PR to isolate the changes

Reply:
ok, I'll look at it.